controllers/util/cephutil.go

```diff
@@ -0,0 +1,18 @@
+package util
+
+import "math"
+
+const (
+	PGCountPerOSD = 600
+	MaxPGUnitSize = 128
```

Comment:
These constants are not used outside of this file. Please make them private
```suggestion
	pgCountPerOSD = 600
	maxPGUnitSize = 128
```